primary/tests/nodes_bootstrapping_tests.rs

```diff
@@ -75,13 +73,13 @@ async fn test_node_staggered_starts() {
 
     tokio::time::sleep(node_staggered_delay).await;
 
-    let rounds = authorities_current_round(&cluster).await;
+    let rounds = authorities_latest_commit_round(&cluster).await;
     assert_eq!(
         rounds.len(),
         4,
-        "Expected to have received metrics from all nodes"
+        "Expected to have received commit metrics from all nodes"
     );
-    assert!(rounds.values().all(|v| v > &1.0), "We have no unavailable nodes, so all should have made progress more than just the first round");
+    assert!(rounds.values().all(|v| v > &1.0), "We have only (f) unavailable nodes, so all should have made progress and committed at least after the first round");
```

Comment:
See my comment above on style for the comparison. Moreover: do you still have any unavailable nodes then?

Reply:
Ah no unavailable nodes, apologies, bad copy/paste :-)